docs/commands/config.md

```diff
@@ -27,8 +28,8 @@ gef➤  gef save
 [+] Configuration saved to '/home/vagrant/.gef.rc'
 ```
 
-Upon startup, if `gef` finds a file `${HOME}/.gef.rc`, it will automatically
-loads its values.
+Upon startup, if either of `$GEF_RC` or `${HOME}/.gef.rc` point to an existing file,
```

Comment:
nit: wording. First points to a file, second is a file.

Reply:
fixed.